contracts/QuestFactory.sol

```diff
@@ -541,24 +541,11 @@ contract QuestFactory is Initializable, OwnableUpgradeable, AccessControlUpgrade
         ++currentQuest.numberMinted;
         questContract_.singleClaim(msg.sender);
 
-        if (mintFee > 0) processMintFee(ref_, currentQuest.mintFeeRecipient);
+        if (mintFee > 0) processMintFee(ref_, currentQuest.questCreator);
```

Comment:
This struct param is unused now - if the recipient is always the creator we can just remove the setter for that in the quest create. I also think we can remove 
```
        if (mintFeeRecipient_ == address(0)) {
            mintFeeRecipient_ = defaultMintFeeRecipient;
        }
        ```
        Since creator is never 0 but worth double checking that...